onegov/election_day/formats/election/wabstic_majorz.py

```diff
@@ -46,15 +47,16 @@
 )
 
 
-def line_is_relevant(line, number, district=None):
-    if district:
-        return line.sortwahlkreis == district and line.sortgeschaeft == number
-    else:
-        return line.sortgeschaeft == number
-
-
-def get_entity_id(line, entities):
-    entity_id = int(line.bfsnrgemeinde or 0)
+def get_entity_id(line):
+    col = 'bfsnrgemeinde'
+    # try:
```

Comment:
Commented out code should not be committed, unless it is meant to be activated in the near future (for example, if a Puppet deployment has been prepared and tested, but not yet rolled out).